actions/pyproject.toml

```diff
@@ -17,14 +17,12 @@ robocorp-log = ">=2.4,<3"
 robocorp-truststore= ">=0.8.0"
 psutil = "^5.0"
 docstring_parser_fork = "^0.0.5"
+pydantic = "^2.0" # Required for the `Response[T]`
```

Comment:
Left a Slack [thread](https://sema4ai.slack.com/archives/C06CYLQ7S4R/p1716448556918039?thread_ts=1716382901.166559&cid=C06CYLQ7S4R).

I might be off-topic, just an idea.